zerver/lib/bulk_create.py

```diff
@@ -63,6 +66,7 @@ def bulk_create_users(realms, users_raw, bot_type=None):
     Subscription.objects.bulk_create(subscriptions_to_create)
 
 def bulk_create_streams(realms, stream_list):
+    # type: (Dict[str, Realm], Iterable[Tuple[Any, Any]]) -> None
```

Comment:
When I replaced `Any` with `text_type` I got
```
zerver/lib/bulk_create.py:76: error: Invalid index type "unicode" for "dict"
```
which I'm pretty sure happens because the first item from the tuple is used as the key for the first argument which is `Dict[str, Realm]`.

Replacing both of the `Any` arguments with `str` seems to work so it looks like that's the way to go.